.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
I would agree if we had that much Issues, but we don't, really. On the first page we have an issue from December last year, so in my opinion 10/30 will be too short.